content/blog/2025-11-03-urssi-fall-fellowship-ai-in-sci-soft.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "URSSI Fall Fellowship: Characterizing the Impact of LLM-Generated Code in Scientific Software"
+date: 2025-11-03
+author: "Eva Maxfield Brown, David Farr, and Shahan Ali Memon"
+---
+
+## When AI Starts Writing Scientific Software
+
+As a multi-purpose technology, large language models (LLMs) have rapidly been integrated across industries including academia and software development. From suggesting syntax to writing full functions, they’ve transformed the possibilities for how scientists build and maintain research software.
+
+However, the speed and convenience of LLMs comes with trade-offs. Scientific software often lives at the frontier of discovery, where a subtle bug or misused library can distort an entire analysis[^1]. As researchers increasingly use tools like ChatGPT or Claude to write code, we face a new question:
+
+> How is AI-generated code changing the structure, reliability, and culture of scientific software?
+
+[^1]: [O'Brien (2025)](https://doi.org/10.1145/3706598.3713668) finds that researchers who have used generative AI for code assistance in their work typically rely on verification strategies such as "eyeballing" which ultimately did not work and as a result, the LLMs "introduced unplanned or incorrect elements into research code."
+
+## What We’re Building
+
+We’re launching a six-month study to analyze the impact of AI-assisted code in open-source scientific repositories. Our goal is to produce quantitative and qualitative analyses of scientific software repositories spanning the past decade, documenting several metrics along the way that may point to a change in scientific software associated with the advent of LLMs. 
+
+Our project has three main objectives:
+
+1. **Build a dataset of scientific source code**
+
+We’re curating a dataset of paired scientific publications and their associated source-code repositories which are publicly hosted on GitHub. The source-code repositories include a mix of "analysis repositories" (repositories which store the analysis code and visualization scripts for a publication) and  "research tools" (general purpose libraries and reusable tools typically attached to 'software papers'). In addition, our dataset includes repositories with development activity from 2015 (“pre-Gen AI”) to more recent projects possibly influenced by AI tools, including a subset of repositories that span both Pre-Gen AI and Post-Gen AI time periods.
```

Comment:
minor, but in "Pre-Gen AI" and "Post-Gen AI", "pre" and "post" should not be capitalized